docs/astro/sql/operators/transform.rst

```diff
@@ -45,3 +45,11 @@ Case 2: When we pass a Pandas dataframe while completing a data transformation.
        :language: python
        :start-after: [START transform_example_4]
        :end-before: [END transform_example_4]
+
+Case 3: When we pass SQL file in the transform decorator.
```

Comment:
Wouldn't it make more sense to `return` the file in the function instead of adding another argument to the decorator? What was the reasoning behind that decision?

Reply:
@feluelle Yes, it would. but the code we have suggests we pass it in decorator, I need to confirm this behavior with daniel once.